src/controllers/auth.js

```diff
@@ -19,10 +19,13 @@ module.exports = {
       if (!validPassword) throw new Error('Wrong password');
 
       const payload = {
-        username,
-        email: user.email,
-        firstName: user.firstName,
-        lastName: user.lastName,
+        user: {
+          username: user.username,
+          email: user.email,
+          firstName: user.firstName,
+          lastName: user.lastName,
+          _id: user._id,
+        },
```

Comment:
I had to edit the payload to match the expected verified value in the authentication middleware